newrelic-agent/build.gradle

```diff
@@ -94,6 +94,8 @@ dependencies {
 
     shadowIntoJar('com.google.guava:guava:28.2-jre') 
 
+    shadowIntoJar('com.github.ben-manes.caffeine:caffeine:2.9.0')
```

Comment:
Caffeine 3.x requires Java 11.x; the 2.x line will [remain supported](https://github.com/ben-manes/caffeine/releases/tag/v3.0.0).